baselines/dev/test-baseline-structure.sh

```diff
@@ -6,7 +6,7 @@ if [ "$#" -ne 1 ]; then
     exit 1
 fi
 baseline_dir=$1
-baseline_path=$(pwd)/$1
+baseline_path=$(cd .. && pwd)/$1
```

Comment:
@jafermarq can you review this?

Reply:
@achiverram28 we discussed about this earlier today. I think we concluded this is not needed, right?

Reply:
yes